### PR TITLE
Party Sheet button fix

### PR DESCRIPTION
**Pull request by JustinFreitas**
Puts the button on the search line before the search icon.  Adjust as necessary.

### CODE REVIEW

**Comment by wyrmisis** [comment]
Mind dropping a screenshot for this change?

**Comment by JustinFreitas** [comment]
> Mind dropping a screenshot for this change?

image

**Review by Rughalt — CHANGES_REQUESTED** [review]
Onbe minor change I would like to request - the button seems to miss left margin and it looks off when taking into account the buttons above it.

**Comment by JustinFreitas** [comment]
> Onbe minor change I would like to request - the button seems to miss left margin and it looks off when taking into account the buttons above it.

Yeah, I can get that figured out.  Was the Party Sheet button on the upper button line before or was it always on the line with the search controls?  Anyway, if you remember, let me know... for now, I'll continue with it on the search line.

**Comment by Rughalt** [comment]
Search line is good, lets just make it nice and tidy CSS wise! Thanks!

File: src/scss/apps.scss [hunk 1]
```diff
@@ -241,6 +241,7 @@
     width: 32px;
     text-align: center;
     line-height: 20px;
+    margin: 4px;
```
Comment (JustinFreitas) [inline src/scss/apps.scss:244]: Adding in this margin it now looks like this.

image

**Review by Rughalt — APPROVED** [review]

**Pull request merged**